src/components/TaskDashboard.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
+import { Task } from './types';
 
-function TaskDashboard() {
+interface TaskList {
```

Comment:
```suggestion
interface TaskDashboardProps{
```
> ou ate
> ```suggestion
> interface Props{
> ```
futuramente provavelmente terao mais props e entao esse nome atual perdera o sentido

Reply:
Valeu pela dica.